chia/cmds/wallet.py

```diff
@@ -506,6 +506,14 @@ def nft_wallet_create_cmd(wallet_rpc_port: Optional[int], fingerprint: int) -> N
     show_default=True,
     callback=validate_fee,
 )
+@click.option(
+    "-rp",
+    "--royalty-percentage",
+    help="NFT royalty percentage in basis points. Example: 1750 would represent 1.75%",
```

Comment:
perhaps this is a bit of a nit-pick; but "percent" means hundreds, and this is specified in ten thousandths. using the term "percentage" is odd in my mind. it seems "percentage" here is used to mean "fraction" or "factor".

Reply:
we have updated the naming, thanks